dijet/tasks/jersf.py

```diff
@@ -0,0 +1,294 @@
+# coding: utf-8
+
+"""
+Custom base tasks.
+"""
+
+
+import law
+
+from columnflow.tasks.framework.base import Requirements, BaseTask, ShiftTask
+from columnflow.tasks.framework.mixins import (
+    CalibratorsMixin, SelectorStepsMixin, ProducersMixin, MLModelsMixin,
+    VariablesMixin, DatasetsProcessesMixin, CategoriesMixin,
+)
+# from columnflow.tasks.framework.remote import RemoteWorkflow
+from columnflow.tasks.reduction import MergeReducedEventsUser, MergeReducedEvents
+from columnflow.tasks.production import ProduceColumns
+from columnflow.tasks.ml import MLEvaluation
+from columnflow.tasks.histograms import MergeHistograms
+from columnflow.util import dev_sandbox, maybe_import
+
+from dijet.constants import eta, pt
+
+ak = maybe_import("awkward")
+hist = maybe_import("hist")
+np = maybe_import("numpy")
+plt = maybe_import("matplotlib.pyplot")
+mplhep = maybe_import("mplhep")
+
+
+class HBWTask(BaseTask):
+
+    task_namespace = "dijet"
+
+
+class ColumnsBaseTask(
```

Comment:
I'm curious what the rationale is to have such a base task. It doesn't look like it's being derived from at the moment, but is there a planned use case?

Reply:
Yes, we should follow up on this. My point was that the code doesn't seem like it's currently being used at all. In general it's preferable not to have unused code on master for maintainability reasons. But since we probably plan to develop tasks based on this in the near future I'm OK with leaving it in for now.